apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -193,7 +200,7 @@ type CssValueInputProps = {
    */
   keywords?: Array<KeywordValue>;
   onChange: (value: CssValueInputValue) => void;
-  onChangeComplete: (value: StyleValue) => void;
+  onChangeComplete: (value: StyleValue, reason: ChangeReason) => void;
```

Comment:
Just wrote to Oleg in DM that for esc we need to know the reason of called events.
Moreover seems like I need to distinguish blur and enter too, so it looks like a good change.

Reply:
Might make sense to change the callback type from value to {value, reason} to keep the single-callback pattern